auto_py_to_exe/web/js/i18n.js

```diff
@@ -32,6 +32,7 @@ const translationMap = {
                 fi: 'Skriptin sijainti',
                 vi: 'Vị trí tập lệnh',
                 nl: 'Script Locatie',
+                ko: "스크립트 위치"

```

Comment:
Can you add commas to the end of these new additions? This helps reduce the diff in future changes.

```suggestion
                ko: "스크립트 위치",
```

(more below)